test/jdk/javax/swing/JTable/4275046/bug4275046.java

```diff
@@ -0,0 +1,193 @@
+        table = new JTable(model);
+        table.getColumnModel().getColumn(1).setCellEditor(comboEditor);
+
+        frame.add(table);
+        frame.pack();
+        frame.setSize(550, 400);
+        frame.setVisible(true);
+    }
+
+    private void test() throws Exception {
+        try {
+            SwingUtilities.invokeAndWait(new Runnable() {
+		    @Override
+		    public void run() {
+			createGUI();
+		    }
+	    });
+
+            runTest();
+            checkResult();
+        } finally {
+            SwingUtilities.invokeAndWait(new Runnable() {
+		    @Override
+		    public void run() {
+                        if (frame != null) {
+                            frame.dispose();
+                        }
+		    }
+            });
+        }
+    }
+
+    private void runTest() throws Exception {
+        robot.waitForIdle();
+
+        // Click the first cell in the "color" column
+        SwingUtilities.invokeAndWait(new Runnable() {
+	    @Override
+            public void run() {
+                tableLoc = table.getLocationOnScreen();
+                cellRect = table.getCellRect(0, 1, true);
+            }
+        });
+
+        robot.mouseMove(tableLoc.x + cellRect.x + cellRect.width / 2,
+                        tableLoc.y + cellRect.y + cellRect.height / 2);
+        robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+        robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+        robot.waitForIdle();
+
+        // Edit the cell
+        robot.keyPress(KeyEvent.VK_E);
+        robot.keyRelease(KeyEvent.VK_E);
+
+        robot.keyPress(KeyEvent.VK_D);
+        robot.keyRelease(KeyEvent.VK_D);
+
+        robot.keyPress(KeyEvent.VK_I);
+        robot.keyRelease(KeyEvent.VK_I);
+
+        robot.keyPress(KeyEvent.VK_T);
+        robot.keyRelease(KeyEvent.VK_T);
+
+        robot.keyPress(KeyEvent.VK_E);
+        robot.keyRelease(KeyEvent.VK_E);
+
+        robot.keyPress(KeyEvent.VK_D);
+        robot.keyRelease(KeyEvent.VK_D);
+        robot.delay(100);
+
+        // Click another cell
+        SwingUtilities.invokeAndWait(new Runnable() {
+	    @Override
+            public void run() {
+                cellRect = table.getCellRect(1, 2, true);
+            }
+        });
+
+        robot.mouseMove(tableLoc.x + cellRect.x + cellRect.width / 2,
+                        tableLoc.y + cellRect.y + cellRect.height / 2);
+        robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+        robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+        robot.delay(100);
+    }
+
+    private void checkResult() throws Exception {
+        robot.waitForIdle();
+        SwingUtilities.invokeAndWait(new Runnable() {
+	    @Override
+            public void run() {
+            // Read the edited value of from the cell
+            editedValue = table.getModel().getValueAt(0, 1);
+            System.out.println("The edited value is = " + editedValue);
+	    testResult = editedValue.equals(EXPECTED_VALUE);
```

Comment:
The indentation is wrong: the body of the method must be indented by four spaces.
There are other misindented blocks; maybe it's because of the tabs in the file. Please correct indentation.